### ISSUE TITLE
Filter filters every post instead of just OP

### ISSUE THREAD

**Issue by OpenUp596** [Floens/Clover#377]
Instead of just highlighting/hiding threads in the catalog, filters starts to highlight/hide every single post that matches a certain pattern. 

If i uncheck "comment" in my filters it stops doing it but it also stops filtering the thread if the key word appears in OP's comment. 

Clover version: 2.3.1

**Comment by Floens** [comment]
I agree that's it's confusing. The filters now are always for all posts.

The action is what action is taken, but it's always taken on all posts, there is no control over just OPs or everything. We'll have to add that.

The comment means that it'll look for matches within the comment.
